cmd/boulder-sa/main.go

```diff
@@ -58,16 +62,36 @@ func main() {
 	dbMap, err := sa.NewDbMap(dbURL, saConf.DBConfig.MaxDBConns)
 	cmd.FailOnError(err, "Couldn't connect to SA database")
 
+	sa.SetSQLDebug(dbMap, logger)
```

Comment:
Can you explain this addition? I don't see the connection with the gRPC work at face-value.